src/rgw/rgw_data_sync.cc

```diff
@@ -729,7 +734,8 @@ int RGWRemoteDataLog::read_sync_status(const DoutPrefixProvider *dpp, rgw_data_s
   RGWDataSyncCtx sc_local = sc;
   sc_local.env = &sync_env_local;
 
-  ret = crs.run(dpp, new RGWReadDataSyncStatusCoroutine(&sc_local, sync_status));
+  ret = crs.run(dpp, new RGWReadDataSyncStatusCoroutine(&sc_local, sync_status,
+							nullptr));
```

Comment:
@cbodley  @adamemerson  in light of the recent issue where the sync status command shows 'shards behind' incorrectly (even though we have synced everything), wondering if we are reading the wrong version? I am not very familiar with the internals of the version tracking mechanism, but would it make sense to pass in objv_tracker here?

Reply:
if this `RGWRemoteDataLog::read_sync_status()` function is only used by radosgw-admin, i don't think adding objv_tracker would help anything

Reply:
@cbodley yep I realized that. thanks!